week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,91 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
-
-// The answer should come from shaking the ball
+const number8 = Math.ceil(Math.random() * 20);
```

Comment:
There's an interesting edge-case here :) `Math.random()` may return `0` (it's very unlikely, but it can happen), and `Math.ceil(0)` is `0`. That won't hit your `if` branches because they start at `1`.

You probably want to use `Math.floor(Math.random() * 20)` and start your checks at `0`, rather than using `ceil` and starting at `1` :)